packages/pf-codemods/README.md

```diff
@@ -192,6 +192,24 @@ Out:
 <CodeEditor      />
 ```
 
+### clipboardCopy-remove-switchDelay [(#8619)](https://github.com/patternfly/patternfly-react/pull/8619)
+
+We've removed the switchDelay prop the `ClipBoardCopy` component.
```

Comment:
```suggestion
We've removed the switchDelay prop from the `ClipBoardCopy` component.
```

Reply:
man idk how i ended up with so many typos on this one